readthedocs/doc_builder/environments.py

```diff
@@ -360,8 +360,7 @@ def get_wrapped_command(self):
             ])
         )
         return (
-            "/bin/sh -c 'cd {cwd} && {prefix}{cmd}'".format(
-                cwd=self.cwd,
+            "/bin/sh -c '{prefix}{cmd}'".format(
                 prefix=prefix,
```

Comment:
Think we may also have one here in the prefix, I'm looking into it. But same as this one, this isn't a vulnerability.

Reply:
Escaped!